src/styles/base/_typography.scss

```diff
@@ -1,6 +1,6 @@
 @use "../abstracts/variables" as *;
 
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=Jost:wght@700&family=Lato:ital@1&family=Mulish:wght@600&family=Poppins:wght@400;500;600;700&family=Space+Mono:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=Jost:wght@700&family=Lato:wght@100;300;400;700;900&family=Mulish:wght@600&family=Poppins:wght@400;500;600;700&family=Space+Mono:wght@700&display=swap");
```

Comment:
what was this change for?

Reply:
We changed the font family fron Italic to normal, you can notice in the figma, it's using the normal font where in the link. The part &family=Lato:ital only gets fonts with italic properties. We needed normal fonts